src/Common/src/Interop/Windows/Kernel32/Interop.GlobalLock.cs

```diff
@@ -9,10 +9,16 @@ internal partial class Interop
 {
     internal partial class Kernel32
     {
-        [DllImport(Libraries.Kernel32, PreserveSig = true)]
-        internal static extern IntPtr GlobalLock(IntPtr hMem);
+        [DllImport(Libraries.Kernel32, ExactSpelling = true)]
+        public static extern IntPtr GlobalLock(IntPtr hMem);
 
-        [DllImport(Libraries.Kernel32, PreserveSig = true)]
-        internal static extern int GlobalUnlock(IntPtr hMem);
+        [DllImport(Libraries.Kernel32, ExactSpelling = true)]
+        public static extern IntPtr GlobalLock(HandleRef hMem);
+
+        [DllImport(Libraries.Kernel32, ExactSpelling = true)]
+        public static extern IntPtr GlobalUnlock(IntPtr hMem);
+
+        [DllImport(Libraries.Kernel32, ExactSpelling = true)]
+        public static extern IntPtr GlobalUnlock(HandleRef hMem);
```

Comment:
SetLastError = true is being removed from a bunch of these.  Does that mean we never check the error code after calling any of these?